hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/WithScmClient.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.scm.cli.container;
+
+import org.apache.hadoop.hdds.scm.client.ScmClient;
+
+/**
+ * Command which provides and SCM client based on the current config.
+ */
+public interface WithScmClient {
```

Comment:
Since you renamed `scmcli` to `admin`, I feel that OzoneAdmin or Admin would be a more appropriate name for the interface. Thoughts?

Reply:
Based on the function, this interface nothing more just providing an SCM client. Yes, I agree that it's used by admin commands but I am not sure if it's important here. (It can be used for any other command, eg. with Freon)